### ISSUE TITLE
Auto discovering phonegap serve - features

### ISSUE THREAD

**Issue by gabrielstuff** [phonegap/connect-phonegap#140]
Using mdns or other module it could be easy to implement bonjour / zero-conf auto discovering and allow autoconnecting to server apps.

**Comment by mwbrooks** [comment]
I want this so bad!

Related to: phonegap/phonegap-app-developer#97

**Comment by gabrielstuff** [comment]
Indeed. We've been using https://github.com/agnat/node_mdns but some issue occurs with multiple services per ip appearing. We manage to filter them on client side.
The good part is that the bonjour plugin works perfectly fine on iOS ! https://github.com/vstirbu/ZeroConf 
It could be pretty fast to implement, but the method + design has to be well thought. 

Btw what's the place to submit design proposal if so ?

Thanks
